main.go

```diff
@@ -94,6 +94,39 @@ func (s *server) GetTemperatureAndHumidity(ctx context.Context,
 	}, e
 }
 
+func (s *server) ReadEncoders(ctx context.Context,
+	in *pb.ReadEncodersRequest) (*pb.ReadEncodersResponse, error) {
+	var leftFront, leftBack, rightFront, rightBack int32
+	var e error
+	if leftFront, e = motors.ReadEncoder(mc.EncoderLeftFront); e != nil {
+		return &pb.ReadEncodersResponse{
+			Status: &pb.Status{},
```

Comment:
This will return successful status, but I think we need to return an error here.